packages/test-utils/make-scroll.ts

```diff
@@ -1,8 +1,9 @@
 import { sleep } from '@element-plus/test-utils'
 
 const makeScroll = async (dom: Element, name: 'scrollTop' | 'scrollLeft', offset: number) => {
+  const eventTarget = dom === document.documentElement ? window : dom
   dom[name] = offset
-  dom.dispatchEvent(new CustomEvent('scroll'))
+  eventTarget.dispatchEvent(new CustomEvent('scroll'))
```

Comment:
Because `new CustomEvent()` can not be used in IE11 and vue 3 resease version will be compatible with IE11, so I think should be judged whether it is IE11 for compatibility.

![image](https://user-images.githubusercontent.com/33176053/92377574-ec1be300-f136-11ea-95f7-1a7d237c7177.png)

Reply:
> Because `new CustomEvent()` can not be used in IE11 and vue 3 resease version will be compatible with IE11, so I think should be judged whether browser is IE11 for compatibility.
> 
> ![image](https://user-images.githubusercontent.com/33176053/92377574-ec1be300-f136-11ea-95f7-1a7d237c7177.png)

You're right, I will raise another pr to fix it later

Reply:
Sorry, I was out of my mind earlier. It's no need to worry about the compatibility for that it just a mock in test. @Ryan2128

Reply:
YES, I also didn't notice it.